addons/crm/static/src/js/tours/crm.js

```diff
@@ -39,11 +39,11 @@ tour.register('crm_tour', {
     content: Markup(_t("Now, <b>add your Opportunity</b> to your Pipeline.")),
     position: "bottom",
 }, {
-    trigger: ".o_opportunity_kanban .o_kanban_group:first-child .o_kanban_record:last-child .oe_kanban_content",
+    trigger: ".o_opportunity_kanban .o_kanban_group:first-child .o_kanban_record:last-of-type .oe_kanban_content",
```

Comment:
About this change: `last-child` only works if it is the actual _last_ child, there were some changes to the dom and now after the kanban_records there is a "button" to create new records as well.